src/main/java/org/folio/innreach/external/client/feign/error/InnReachFeignErrorDecoder.java

```diff
@@ -20,6 +21,11 @@ public Exception decode(String methodKey, Response response) {
       return new BadCredentialsException("Can't get InnReach access token. Key/Secret pair is not valid");
     }
 
+    if (HttpStatus.valueOf(response.status()).equals(HttpStatus.SERVICE_UNAVAILABLE)) {
+      log.debug("CentralServer is suspended for contribution");
+      return new ServiceSuspendedException("CentralServer is suspended for contribution. Try again later");
+    }
+
```

Comment:
This is good, but I think this is only going to handle the exception if we are using version 3. We also need to check for the string that Justin provided in the response body for version 2.

Reply:
That part is handled in InnReachResponse object where we are getting 200 response while contributing & decontributing

Reply:
But why are handling this in two different places? I understand that one is a response code and one is a response body, but they are both part of the same response. Shouldn't error handling be consolidated in one place? See my comments below as well about code duplication.